src/storyMap/components/StoryMapsHomeCardDefault.js

```diff
@@ -32,47 +28,27 @@ const StoryMapsHomeCardDefault = () => {
 
   return (
     <HomeCard
-      aria-labelledby="story-maps-default-title"
-      sx={{ flexDirection: 'column' }}
+      id="story-maps"
+      isDefault
+      title={t('storyMap.home_title')}
+      action={{
+        label: t('storyMap.home_create'),
+        to: 'tools/story-maps/new',
+      }}
+      image={{
+        src: storyMapImage,
+        to: '/tools',
+        alt: t('tool.home_card_img_alt'),
+      }}
     >
-      <Box sx={{ p: 2 }}>
-        <Typography
-          id="story-maps-default-title"
-          variant="h2"
-          sx={{ pt: 0, mb: 4 }}
-        >
-          {t('storyMap.home_title')}
-        </Typography>
-
-        <Stack direction="row" spacing={3}>
-          <Link component={RouterLink} to="/tools">
-            <Paper
-              variant="outlined"
-              component="img"
-              src={storyMapImage}
-              alt={t('tool.home_card_img_alt')}
-              height={64}
-            />
+      <Box sx={{ display: 'flex', alignItems: 'center', pr: 1 }}>
+        <Typography variant="body1">
+          {t('storyMap.home_default_description')}
+          <Link href="/tools/story-maps" sx={{ textDecoration: 'underline' }}>
+            {t('storyMap.home_default_description_get_inspired')}
           </Link>
```

Comment:
issue: The period should be after the hyperlink. You'll probably want to use a `<Trans>` component to do this.